rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -19,6 +21,18 @@
  */
 public final class RxView {
   /**
+   * Create an observable of attach and detach events on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult
+  public static Observable<ViewAttachEvent> attachEvents(View view) {
```

Comment:
I have an idea of how it could be done in a simple way, let me tinker for a min and get back to you